src/test/java/app/bpartners/geojobs/unit/FileUnzipperTest.java

```diff
@@ -0,0 +1,47 @@
+package app.bpartners.geojobs.unit;
+
+import static app.bpartners.geojobs.file.FileZipper.ZIP_FILE_SUFFIX;
+import static java.time.Instant.now;
+import static org.junit.Assert.assertThrows;
+
+import app.bpartners.geojobs.conf.FacadeIT;
+import app.bpartners.geojobs.file.FileUnzipper;
+import java.io.*;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+import java.util.zip.ZipOutputStream;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+class FileUnzipperTest extends FacadeIT {
+  @Autowired private FileUnzipper fileUnzipper;
+
+  private static final String FILE_NAME =
+      "src"
+          + File.separator
+          + "test"
+          + File.separator
+          + "resources"
+          + File.separator
+          + "mockData"
+          + File.separator
+          + "zipFile.zip";
+
+  ZipFile zipFile() throws IOException {
```

Comment:
bad name. You are not constructing a standard zipFile. You are constructing a MALICIOUS zipFile (which contains path traversal attack with ".."). Make it explicit in the name